lib/sqlalchemy/sql/schema.py

```diff
@@ -4485,9 +4538,14 @@ def _reload(self, columns: Iterable[Column[Any]]) -> None:
         PrimaryKeyConstraint._autoincrement_column._reset(self)  # type: ignore
         self._set_parent_with_dispatch(self.table)
 
-    def _replace(self, col: Column[Any]) -> None:
-        PrimaryKeyConstraint._autoincrement_column._reset(self)  # type: ignore
-        self._columns.replace(col)
+    def _replace(self, col: Union[Column[Any], Period]) -> None:
```

Comment:
is this ever called?   _replace() is a very special method used by the ORM, I dont think there are any cases where this would be called with a Period